pyanaconda/modules/payloads/source/live_os/initialization.py

```diff
@@ -108,11 +108,37 @@ def run(self):
         return SetupLiveOSResult(required_space=required_space)
 
     def _calculate_required_space(self):
-        """Calculate size of the live image image."""
-        source = os.statvfs(self._target_mount)
-        required_space = source.f_frsize * (source.f_blocks - source.f_bfree)
-        log.debug("Required space: %s", Size(required_space))
-        return required_space
+        """
+        Calculate the disk space required for the live OS by summing up
+        the size of relevant directories using 'du -s'.
+        """
+        exclude_patterns = [
+            "/dev/",
+            "/proc/",
+            "/tmp/*",
+            "/sys/",
+            "/run/",
+            "/boot/*rescue*",
+            "/boot/loader/",
+            "/boot/efi/loader/",
+            "/etc/machine-id",
+            "/etc/machine-info"
+        ]
+
+        # Build the `du` command
+        du_cmd_args = ["-s", "--bytes", "--summarize", self._target_mount]
```

Comment:
Based on the man page the `-s` is `--summarize` so you are specifying the same flag twice.